src/pages/Home.js

```diff
@@ -1,7 +1,8 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
 
-import { Wrapper } from '../components/GlobalStyles';
+import { Wrapper } from "../components/GlobalStyles";
+import { SearchBox } from "../components/SearchBox";
```

Comment:
Not sure why you changed the quotes :)

Reply:
`prettierrc` changed the quotes.

Reply:
It's strange, it's single in here: https://github.com/codesect/cs01-team-grace-01/blob/cdf3f99cba6df5a0f458df68a56d255e2ef3cb77/.prettierrc#L11

Reply:
in `master` branch it is applyign single quotes but not in `feature/seachboxform`  branch